docs/semgrep-supply-chain/triage-remediation.md

```diff
@@ -94,17 +94,53 @@ To assess your findings, Semgrep Supply Chain provides the following methods:
-| **Severity**           | Filter by the severity of a finding. Filters are based on the severity of a vulnerability. Semgrep Supply Chain rules use severity values set by the GitHub Advisory Database.  |
-| **Transitivity**       | Filter by the transitivity of the finding. <ul><li>**Direct**: Your project depends directly on the dependency.</li><li>**Transitive**: Your project's dependency depends on a vulnerable dependency.</li><li>**Undetermined**: Semgrep had no transitivity information for the dependency as it relates to your project.</li></ul> |
-| **EPSS probability**   | [Exploit prediction scoring system (EPSS) probability](https://www.first.org/epss/). Represents the likelihood that the vulnerability will be exploited in the wild in the next 30 days. Values range from 0% to 100% where the higher the score, the greater the probability the vulnerability is exploited. Semgrep groups probabilities as follows: <ul><li><b>High</b>: 50 - 100%</li><li><b>Medium</b>: 10 - &#60;50%</li><li><b>Low</b>: &#60;10% </li></ul> |
-| **Reachability**       | Filter by exposure, or whether the finding is reachable or not. <ul><li>**Reachable**: A finding is reachable if there's a code pattern in the codebase that matches the vulnerability definition</li><li>**Always reachable**: A finding is always reachable if it's something Semgrep recommends fixing, regardless of what's in the code.</li><li>**Conditionally reachable**: A finding is conditionally reachable if Semgrep finds a way to reach it when scanning your code when certain conditions are met.</li><li>**No Reachability Analysis**: A finding that isn't scanned by Semgrep for reachability.</li><li>**Unreachable**: A finding is unreachable if you don't use the vulnerable piece of code of the imported library or package.</li></ul>|
-| **Dependencies**         | Filter for findings based on the name of the dependency involved. |
-| **Rules**                | Filter based on the rule that generated the finding. |
+| Filter | Description  |
+| -  | - |
+| **Projects and branches** | The repositories connected to your Semgrep account and by findings in different Git branches. |
+| **Tags** | The tags associated with the project. |
+| [**Status**](#status) | The triage state of a finding. |
+| **Severity** | The severity of a finding. Filters are based on the severity of a vulnerability. Semgrep Supply Chain rules use severity values set by the [GitHub Advisory Database](https://github.com/advisories).  |
+| [**Transitivity**](#transitivity) | The transitivity of the finding. |
+| [**EPSS probability**](#epss-probability) | The finding's [Exploit prediction scoring system (EPSS) probability](https://www.first.org/epss/). |
+| [**Reachability**](#reachability) | The finding's exposure, or whether it is reachable or not. |
```

Comment:
Does reachability here refer to reachable & always reachable? I'm not sure I recall which of those "reachable" states applh.

Reply:
@s-santillan yep, it includes those two plus more:

<img width="188" alt="image" src="https://github.com/user-attachments/assets/90deabb7-1719-4cd5-ab54-dad85d5c33d7">

Do you think I need more clarification here?